unix/irc_client_unix.ml

```diff
@@ -11,6 +11,7 @@ module Io_unix = struct
     let sock = Unix.socket Unix.PF_INET Unix.SOCK_STREAM 0 in
     let sockaddr = Unix.ADDR_INET (addr, port) in
     Unix.connect sock sockaddr;
+    Unix.set_nonblock sock;
```

Comment:
This doesn't seem to do what the commit message says it does - shouldn't it just be `Unix.setsockopt sock Unix.SO_KEEPALIVE true` ?

Reply:
my bad, I kept an obsolete commit message. keepalive shouldn't be needed since IRC has its own `ping/pong` mechanism.